pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -140,16 +140,16 @@ let
     src = fetchFromGitHub {
       owner = "opencv";
       repo = "opencv_3rdparty";
-      rev = "0cc4aa06bf2bef4b05d237c69a5a96b9cd0cb85a";
-      hash = "sha256-/kHivOgCkY9YdcRRaVgytXal3ChE9xFfGAB0CfFO5ec=";
+      rev = "7f55c0c26be418d494615afca15218566775c725";
+      hash = "";
```

Comment:
@GaetanLepage  sorry for being late, but the hash being empty seems not right to me. Did you build with `enableIpp = true`? 

Also when looking at https://github.com/opencv/opencv/blob/4.11.0/3rdparty/ippicv/ippicv.cmake#L7, I see a different rev for darwin.  I think we should honor that here too.

Reply:
Well, we could use a `if` or `mkIf` construct to change the hash according to the plateform, but we'd need a contributor with a mac to test (I don't have that :-/).